test/inline-compare/compare.py

```diff
@@ -33,8 +33,10 @@ def __init__(self, image, report_dir="./"):
 
     def packages(self):
         python_packages, python_metadata = self._python_packages()
-        os_pacakges, os_metadata = self._os_packages()
-        return python_packages | os_pacakges, {**python_metadata, **os_metadata}
+        npm_packages, npm_metadata = self._npm_packages()
```

Comment:
Shouldn't this npm_packages work be separated into its own PR to keep traceability clean?

Reply:
I can split the other two into another PR 👍